woocommerce-gateway-ebanx/gateways/class-wc-ebanx-global-gateway.php

```diff
@@ -47,7 +47,11 @@
 		'credit_card_instalments' => '1',
 		'due_date_days' => '3',
 		'brazil_taxes_options' => 'cpf',
-		'interest_rates_enabled' => 'no'
+		'interest_rates_enabled' => 'no',
+		'min_instalment_value_brl' => '20',
+		'min_instalment_value_usd' => '20',
+		'min_instalment_value_eur' => '20',
```

Comment:
20 dollars and 20 euros are not restricted.
Those could default to 0 as far as we know.